ReactWindows/ReactNative.Shared/Views/Text/FontStyleHelpers.cs

```diff
@@ -0,0 +1,70 @@
+﻿#if WINDOWS_UWP
+using Windows.UI.Text;
+#else
+using System.Windows;
+#endif
+
+namespace ReactNative.Views.Text
+{
+    static class FontStyleHelpers
+    {
+        public static FontWeight? ParseFontWeight(string fontWeightString)
+        {
+            var fontWeightNumeric = fontWeightString != null
+                ? ParseNumericFontWeight(fontWeightString)
+                : -1;
+
+            if (fontWeightNumeric > ushort.MaxValue)
+            {
+                return FontWeights.ExtraBold;
+            }
+            else if (fontWeightNumeric > 0)
+            {
+                return ConvertNumeric(fontWeightNumeric);
+            }
+            else if (fontWeightString == "bold")
+            {
+                return FontWeights.Bold;
+            }
+            else if (fontWeightString == "normal")
+            {
+                return FontWeights.Normal;
+            }
+
+            return null;
+        }
+
+        private static int ParseNumericFontWeight(string fontWeightString)
+        {
+            var result = default(int);
+            if (int.TryParse(fontWeightString, out result))
+            {
+                return result;
+            }
+
+            return -1;
+        }
+
+        private static FontWeight ConvertNumeric(int fontWeightNumeric)
+        {
+#if WINDOWS_UWP
+            return new FontWeight
+                {
+                    Weight = (ushort)fontWeightNumeric,
+                };
+#else
+            if (fontWeightNumeric >= 0 && fontWeightNumeric <= 149) { return FontWeights.Thin; }
```

Comment:
>149 [](start = 63, length = 3)

probably would make more sense to just do `fontWeightNumeric < 150`, that way, if for whatever reason FontWeights moved from discrete to continuous, we'd be ready :-)